src/Platforms/PostgreSQL100Platform.php

```diff
@@ -9,6 +9,9 @@
 
 /**
  * Provides the behavior, features and SQL dialect of the PostgreSQL 10.0 database platform.
+ *
+ * @deprecated This class will be merged with {@see PostgreSQLPlatform} in 4.0 because support for Postgres
```

Comment:
Should it be a `{@link ...}` instead?

Reply:
🤔 

We use both in our codebase. According to the PHPDoc spec, `@see` is correct for referencing other structural elements while `@link` is meant for URLs only.

* https://docs.phpdoc.org/guide/references/phpdoc/tags/link.html
* https://docs.phpdoc.org/guide/references/phpdoc/tags/see.html